EIPS/eip-5516.md

```diff
@@ -1,37 +1,35 @@
 ---
 eip: 5516
 title: Soulbound Multi-Owned Tokens
-description: An interface for non-transferrable, Multi-owned NFTs binding to Ethereum accounts
+description: An interface for non-transferable, Multi-owned NFTs binding to Ethereum accounts
 author: Lucas Martín Grasso Ramos (@LucasGrasso), Matias Arazi (@MatiArazi)
 discussions-to: https://ethereum-magicians.org/t/EIP-5516-soulbound-multi-token-standard/10485
-status: Draft
+status: Review
 type: Standards Track
 category: ERC
 created: 2022-08-19
 requires: 165, 1155
 ---
 
 ## Abstract
-This EIP proposes a standard interface for non-fungible double signature Soulbound multi-tokens. Previous account-bound token standards face the issue of users losing their account keys or having them rotated, thereby losing their tokens in the process. This EIP provides a solution to this issue and allows for the recycling of SBTs: minting of NFTs with the same characteristics for it being transferred to different addresses is no longer necessary.
+This EIP proposes a standard interface for non-fungible double signature Soulbound multi-tokens. Previous account-bound token standards face the issue of users losing their account keys or having them rotated, thereby losing their tokens in the process. This EIP provides a solution to this issue that allows for the recycling of SBTs.
 
 ## Motivation
 This EIP was inspired by the main characteristics of the [EIP-1155](./eip-1155.md) token and by articles in which benefits and potential use cases of Soulbound/Accountbound Tokens (SBTs) were presented. 
 This design also allows for batch token transfers, saving on transaction costs. Trading of multiple tokens can be built on top of this standard and it removes the need to approve individual token contracts separately. It is also easy to describe and mix multiple fungible or non-fungible token types in a single contract.
 
 ### Characteristics
 - The NFT will be non-transferable after the initial transfer
-- Partially compatible with [EIP-1155](./eip-1155.md).
-- Double Signature.
-- Multi-Token.
-- Multi-Owned.
-- Semi-Fungible.
+- Partially compatible with [EIP-1155](./eip-1155.md)
+- Double Signature
+- Multi-Token
+- Multi-Owner
```

Comment:
[Editorial] Consider keep it consistent for using either Muti-Owned or Multi-Owner. This is a word using in the title. If it's intentional to use different word, I'd suggest explain the two individually to avoid reader confusion

Reply:
I was the one that made that change. "Multi-Owner" is a characteristic of a standard, whereas "Multi-Owned" is a characteristic of an instance. Just a semantic thing.